### DISCUSSION TITLE
A problem about "mkdocs serve --no-livereload"

### DISCUSSION THREAD

**Discussion by linxianmiao** [mkdocs/mkdocs#2571] [Q&A] [answered]
![picture](https://user-images.githubusercontent.com/83006176/133397129-67aaf028-33c7-4a10-8c47-eefc4fd0c61d.png)
can I disable this file(livereload.js)?

**Comment by oprypin** [comment, marked as answer]
Sorry, in the current implementation `mkdocs serve` will inject this JavaScript no matter what. `--no-livereload` only makes it so the reload never actually triggers. To get a very similar (still very bad, not suitable for production) server, you can use `mkdocs build && cd site && python3 -m http.server`

**Reply by linxianmiao** [reply]
Thanks for your help. I will try to learn `http.server`. By the way, could you please tell me if you will change the implementation?
My English is not good, sorry.

**Reply by oprypin** [reply]
The implementation will not change at this time. I wrote on your opened issue that first I'd like to hear good arguments why the current implementation is not appropriate.
